coderd/workspaces.go

```diff
@@ -654,7 +654,10 @@ func createWorkspace(
 			// we fall back to creating a new workspace. Otherwise, propagate the unexpected error.
 			if err != nil &&
 				!errors.Is(err, prebuilds.ErrNoClaimablePrebuiltWorkspaces) &&
-				!errors.Is(err, prebuilds.ErrAGPLDoesNotSupportPrebuilds) {
+				!errors.Is(err, prebuilds.ErrAGPLDoesNotSupportPrebuiltWorkspaces) {
+				api.Logger.Error(ctx, "failed to claim prebuilt workspace", slog.Error(err),
+					slog.F("workspace_name", req.Name), slog.F("template_version_preset_id", req.TemplateVersionPresetID))
```

Comment:
No I meant we should log when the error is of one of the expected types.

Reply:
1. maybe let's log in both cases? Because if unexpected error is happened it also make sense to log? Or it will be logged on another level because handler failed?
2. do you want to log it with error or warning? Because `ErrNoClaimablePrebuiltWorkspaces` is not an actual error? It can be normal situation.

Reply:
Sounds reasonable. Warning for expected cases, error for unexpected 👍

Reply:
@dannykopping it became too wordy, but I guess it's okay?